app/assets/stylesheets/application.css.scss

```diff
@@ -160,3 +160,19 @@ i.fa.medium {
 ::-webkit-input-placeholder {
   color: $placeholder-text;
 }
+
+// makes vimeo videos responsive
+.video-responsive {
```

Comment:
We've got several videos on the site but this change is only for one and using a different method. 

Can you check if the other videos are responsive (`home.html.erb` has one) and if they are use the same method?